client/html/templates/checkout/standard/process-body-standard.php

```diff
@@ -172,6 +173,9 @@
 		<?php endforeach; ?>
 	</ul>
 
+    <?php //Custom html from Provider ?>
+    <?=$html;?>
```

Comment:
You should combine these lines into one:
`<?= $this->get( 'standardHtml', '' ); // custom HTML from provider ?>`

Reply:
Okay, done